lesson_02/script.js

```diff
@@ -0,0 +1,14 @@
+"use strict";
+
+const ul = document.getElementById("js-target");
+const li = document.createElement("li");
+const a = document.createElement("a");
+a.text = "これです";
+a.href = "1.html";
+
+const img = document.createElement("img");
+img.src = "bookmark.png";
+img.alt = "ブックマーク";
```

Comment:
```suggestion
const ul = document.getElementById("js-target");
const li = document.createElement("li");
const a = document.createElement("a");
const img = document.createElement("img");

a.text = "これです";
a.href = "1.html";
img.src = "bookmark.png";
img.alt = "ブックマーク";
```
I can summarize it further.